Server/JokesOnYou.Web.Api/Exceptions/WrongCredentialsException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace JokesOnYou.Web.Api.Exceptions
+{
+    public class WrongCredentialsException : Exception
```

Comment:
Im being quite picky here but naming this BadCredentialsException might be more appropiate